documentation/staging/content/faq/external-clients.md

```diff
@@ -124,11 +132,16 @@ that call the resource are located within the cluster, then:
     * [Set up an HTTP tunneling-enabled custom channel](#adding-a-weblogic-custom-channel) on the external WebLogic Servers.
     * Specify URLs on the source server that resolve to the load balancer's address and that start with `http` instead of `t3`.
     * Ensure the load balancer configures the HTTP flow to be 'sticky'.
+    * Configure a WebLogic load balancer method that provides server affinity for the cluster's default load balancing algorithm to enable appropriate routing of requests via replica-aware stubs.
```

Comment:
1 - Please add a link to the new section for changing the algorithm.

2 - "replica-aware stub" is an internal term IMO - so I recommend changing "
to enable appropriate routing of requests via replica-aware stubs" to
something like "to speedup connection creation for EJB and JMS clients."